src/main/java/life/qbic/App.java

```diff
@@ -37,12 +38,34 @@ public static void main(String[] args) throws IOException {
    */
   private static QbicDataDownloader loginToOpenBIS(
       PostmanCommandLineOptions commandLineParameters) {
-    LOG.info(String.format("Please provide password for user '%s':", commandLineParameters.user));
 
-    String password = OpenBISPasswordParser.readPasswordFromConsole();
+    String password = "";
+
+    //Password read in from environment Variable
+    if ((!(commandLineParameters.environmentVariableName.isEmpty()))) {
```

Comment:
This is equivalent to
```suggestion
    if (!commandLineParameters.environmentVariableName.isEmpty()) {
```